docroot/modules/custom/mass_content/src/ImageStylesWarmer.php

```diff
@@ -0,0 +1,98 @@
+<?php
```

Comment:
Should we add `declare(strict_types=1);` here, since this is a brand new file?

```suggestion
<?php
declare(strict_types=1);
```